src/topology/sheaves/skyscraper.lean

```diff
@@ -158,4 +169,17 @@ def skyscraper_presheaf_stalk_of_not_specializes_is_terminal
   [has_colimits C] {y : X} (h : ¬p₀ ⤳ y) : is_terminal ((skyscraper_presheaf p₀ A).stalk y) :=
 is_terminal.of_iso terminal_is_terminal $ (skyscraper_presheaf_stalk_of_not_specializes _ _ h).symm
 
+variables [Π (U : opens (Top.of (punit : Type u))), decidable (punit.star ∈ U)]
+
+lemma skyscraper_presheaf_is_sheaf [has_products.{u} C] : (skyscraper_presheaf p₀ A).is_sheaf :=
+(presheaf.is_sheaf_iso_iff (eq_to_iso $ skyscraper_presheaf_eq_pushforward p₀ A)).mpr $
+  sheaf.pushforward_sheaf_of_sheaf _ $ presheaf.is_sheaf_on_punit_of_is_terminal _ $
+    by { dsimp, rw if_neg, exact terminal_is_terminal, exact set.not_mem_empty punit.star }
+
+/--
+A skyscraper presheaf is a sheaf
```

Comment:
```suggestion
The skyscraper presheaf supported at `p₀` with value `A` is the sheaf that assigns `A` to all opens `U` that contain `p₀` and assigns `*` otherwise.
```